forge-gui/res/cardsfolder/upcoming/step_between_worlds.txt

```diff
@@ -0,0 +1,13 @@
+Name:Step Between Worlds
+ManaCost:3 U U
+Types:Sorcery
+K:Plot:4 U U
+A:SP$ GenericChoice | Defined$ Player | TempRemember$ Chooser | Choices$ Stargate,Homebody | ShowChoice$ ExceptSelf | SubAbility$ DBTimetwister | AILogic$ Timetwister | StackDescription$ SpellDescription | SpellDescription$ Each player may shuffle their hand and graveyard into their library. Each player who does draws seven cards. Exile CARDNAME.
+SVar:Stargate:DB$ Pump | Defined$ Remembered | NoteCards$ Self | NoteCardsFor$ Stargate | SpellDescription$ Yes, I want to shuffle my hand and graveyard into my library.
+SVar:Homebody:DB$ Pump | Defined$ Remembered | NoteCards$ Self | NoteCardsFor$ Homebody | SpellDescription$ NO, I DO NOT want to shuffle my hand and graveyard into my library.
+SVar:DBTimetwister:DB$ ChangeZoneAll | ChangeType$ Card.OwnedBy Player.NotedForStargate | Origin$ Hand,Graveyard | Destination$ Library | Shuffle$ True | RememberChanged$ True | UseAllOriginZones$ True | SubAbility$ DBRepeatDraw | StackDescription$ None
```

Comment:
deciding yes is enough
so no extra Remembering needed

Reply:
I take it this means the `DB$ Cleanup` line is unnecessary since I'd assume `TempRemember$ Chooser` would handle the cleanup of the remembered players internally, right?

Reply:
yea that too

Reply:
Works as expected with those edits in 👍